A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_MISC.js

```diff
@@ -28,7 +27,7 @@ class CDU_OPTIONS_MISC {
             if (value !== "") {
                 mcdu.addNewMessage(NXSystemMessages.notAllowed);
             } else {
-                const newPVOption = storedPilotVis.toString();
+                const newPVOption = storedPilotVis ? "0" : "1";
                 NXDataStore.set("CONFIG_PILOT_VISIBILITY", newPVOption);
                 SimVar.SetSimVarValue("L:A32NX_VIS_PILOT_0", "Number", parseInt(newPVOption));
```

Comment:
Let's do it the other way around and cast string to number, rather than reparsing the string as number:

```suggestion
                const newPVOption = storedPilotVis ? 0 : 1;
                NXDataStore.set("CONFIG_PILOT_VISIBILITY", "" + newPVOption);
                SimVar.SetSimVarValue("L:A32NX_VIS_PILOT_0", "Number", newPVOption);
```

Reply:
I dont have any chance to test this for now.